traffic_portal/app/src/common/service/utils/FileUtils.js

```diff
@@ -31,6 +31,56 @@ var FileUtils = function() {
 		a.remove();
 	};
 
+	this.convertToCSV = function(JSONData, reportTitle, includedKeys) {
+		var arrData = typeof JSONData != 'object' ? JSON.parse(JSONData) : JSONData;
+		var CSV = '';
+		CSV += reportTitle + '\r\n\n';
```

Comment:
Any reason for the extra newline, and if so without the carriage return?

Reply:
no real reason. just to give it some space between the title and the body of the report:

![image](https://user-images.githubusercontent.com/251272/103367881-658f6680-4a83-11eb-9878-b39ffd84dffc.png)

Reply:
Cool, just checking. Any reason to not have `\r\n` for the second newline? It doesn't appear to break anything in libreoffice calc or excel, but it's good practice.

Reply:
oh i see. yeah, i can change that.

Reply:
changed